modules/ROOT/pages/sap/sap-connector-reference-51.adoc

```diff
@@ -0,0 +1,657 @@
+| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  | 
+|===
+[[sap_simple-connection-provider]]
+===== Simple Connection Provider
+
+
+Connection provider that connects using a username and password.
+
+
+====== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+| SAP system number a| String |  The two-digit system number (sysnr) of the SAP system. |  | x
+| SAP client ID a| String |  Three-digit client number. Leading zeros are preserved if they appear in the number. |  | x
+| Destination Name a| String |  Identifier of the destination generated by this connector. If none is set, then a random value is created. |  | 
+| Login language a| String |  The language code used by the connector. The language determines the code page used for communicating between the SAP Adapter and the application server. |  EN | 
+| Extended Properties a| Object |  Use to pass in all the database server-specific connection parameters. |  | 
+| Username a| String |  The username the connector uses to log in to SAP. |  | x
+| Password a| String |  The password associated with the login username. |  | x
+| Application server host a| String |  The host of the SAP application server. |  | 
+| Message server configuration a| <<MessageServer>> |  The configuration for message server usage. |  | 
+| Enable RFC tracing on server a| Boolean |  Enable/disable RFC trace on the server. |  false | 
+| Enable CPIC tracing on server a| Enumeration, one of:
+
+** INHERIT
+** NO_TRACING
+** LEVEL_1
+** LEVEL_2
+** LEVEL_3 |  Enable or disable RFC trace on the server. |  | 
+| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  | 
+|===
+
+== Operations
+
+* <<asyncRfc>> 
+* <<confirmTransactionId>> 
+* <<getFunction>> 
+* <<retrieveIdoc>> 
+* <<send>> 
+* <<startsaptransaction>> 
+* <<syncRfc>> 
+
+=== Associated Sources
+* <<document-listener>> 
+* <<function-listener>> 
+
+
+== Operations
+
+[[asyncRfc]]
+=== Asynchronous Remote Function Call
+`<sap:async-rfc>`
+
+
+Executes a BAPIFunction over a queued Remote Function Call (qRFC). A queued RFC is an extension of a transactional RFC (tRFC) that ensures that individual steps are processed in sequence. 
+
+To guarantee that multiple Logical Unit of Work (LUWs) transactions are processed in the order specified by the application. tRFC can be serialized using inbound and outbound queues; hence the name queued RFC (qRFC).  qRFC is best used for:  Extension of the Transactional RFC Defined processing sequence. Use qRFC to guarantee that several transactions are processed in a predefined order.
```

Comment:
Remove the colon and rework that sentence. Maybe just add "the" before "Extension..."